### PR TITLE
feat: added share and tip option on music list three dot button

### LINKED ISSUE #1052: Mobile: Feed Music UI, Button doesn’t work
IMG_2529

### PR DESCRIPTION
**Pull request by sachin-into**
fixes #1052

Screenshot 2024-03-14 at 5 42 55 PM
Screenshot 2024-03-14 at 5 42 36 PM

### CODE REVIEW

File: packages/components/socialFeed/SocialActions/ShareButton.tsx [hunk 1]
```diff
@@ -20,17 +20,25 @@ import { SpacerRow } from "../../spacer";
 interface ShareButtonProps {
   postId: string;
   useAltStyle?: boolean;
+  onSharePress?: () => void;
 }
 
-export const ShareButton = ({ postId, useAltStyle }: ShareButtonProps) => {
-  const [isModalVisible, setIsModalVisible] = useState(false);
+const WEBSITE_URL = "https://app.teritori.com";
```
Comment (omniwired) [inline packages/components/socialFeed/SocialActions/ShareButton.tsx:26]: This is ok in this case, as this will only be used in the mobile app.

**Review by omniwired — DISMISSED** [review]

**Review by n0izn0iz — CHANGES_REQUESTED** [review]
bug: the tip and share modal closes instantly after click on respective button in web

also I feel the ui is not great, maybe sync with konolee?
Screenshot 2024-03-19 at 01 58 09

**Comment by sachin-into** [comment]
changed UI to for now:
Screenshot 2024-03-19 at 11 55 43 AM

**Review by n0izn0iz — APPROVED** [review]
LGTM

**Review by omniwired — APPROVED** [review]

**Pull request merged**